pyomo/contrib/pynumero/sparse/utils.py

```diff
@@ -1,53 +0,0 @@
-#  ___________________________________________________________________________
-#
-#  Pyomo: Python Optimization Modeling Objects
-#  Copyright (c) 2008-2022
-#  National Technology and Engineering Solutions of Sandia, LLC
-#  Under the terms of Contract DE-NA0003525 with National Technology and
-#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain
-#  rights in this software.
-#  This software is distributed under the 3-clause BSD License.
-#  ___________________________________________________________________________
-from scipy.sparse.sputils import isscalarlike
-from scipy.sparse import tril, triu, isspmatrix
-
-import numpy as np
-
-
```

Comment:
Are we going to regret removing these utilities?

Reply:
I highly doubt it. I don't even know where these were originally used. If we ever need them again, it should be easy to get them from the git history.